assets/data/themes/drauven.json

```diff
@@ -322,7 +330,7 @@
 				{
 					"name": "Drauv_ArmR_2handed_lower",
 					"tint": "skin", "tintAmount": 0.4,
-					"depth": 3000,
+					"depth": 8000,
```

Comment:
Did you test to make sure that 8000 isn't too high? A big 2-hander with lots of sticking-out parts should slip in under the fist but over the lower arm, aye?